coresimd/powerpc/altivec.rs

```diff
@@ -0,0 +1,360 @@
+//! PowerPC AltiVec intrinsics.
+//!
+//! AltiVec is a brandname trademarked by Freescale (previously Motorola) for
+//! the standard `Category:Vector` part of the Power ISA v.2.03 specification.
+//! This Category is also known as VMX (used by IBM), and "Velocity Engine" (a
+//! brand name previously used by Apple).
+//!
+//! The references are: [POWER ISA v2.07B (for POWER8 & POWER8 with NVIDIA
+//! NVlink)] and [POWER ISA v3.0B (for POWER9)].
+//!
+//! [POWER ISA v2.07B (for POWER8 & POWER8 with NVIDIA NVlink)]: https://ibm.box.com/s/jd5w15gz301s5b5dt375mshpq9c3lh4u
+//! [POWER ISA v3.0B (for POWER9)]: https://ibm.box.com/s/1hzcwkwf8rbju5h9iyf44wm94amnlcrv
+
+#![allow(non_camel_case_types)]
+
+use coresimd::simd::*;
+use coresimd::simd_llvm::*;
+#[cfg(test)]
+use stdsimd_test::assert_instr;
+
+pub type vector_signed_char = i8x16;
```

Comment:
While it is useful for people porting C code, I'm not so sure we should use the long and confusing types instead of using the common compact ones in the implementation below.

Reply:
The question is, what does the PowerPC C intrinsics specification does? 

For this to hit stable, it helps if "we just do what the PowerPC C intrinsics do in all other compilers or the PowerPC spec/guide".